software_reseller/data/ir_actions_act_window.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <data noupdate="1">
+        <record id="software_reseller.all_licenses_ce314e8b-2439-473a-b7a7-493af7707108" model="ir.actions.act_window" >
```

Comment:
@pimodoo we need some logic for every model

- If the name field exists, then put it in first place.
- If the same model field exists then push after the name (parent-child relation)
.
.
- If any field is related to technical then move to last like (view_id, res_model) 

because the plan is to export the module from the database and make minimal changes.
so it's good to have some logic for every model not just for one.

tell me if I misunderstood something.

Reply:
The goal of the review here is not to have a perfect export tools, it is just to have clean code to maintain. There will always need human review and modifications to have something maintainable and clean.

Your export tool is great, to get a good basis and don't write everything  by hand. So event if it "works" as it is exported, we need to have something more clean to maintain it through versions. These modules will have to be adapted in each version in Odoo when there are model changes. So to ease the job of the one who will adapt future version, we need to have the most consistent code possible.

Reply:
okay